src/app/organisations/data/organisationsStorage.js

```diff
@@ -186,6 +186,14 @@ const pagedSearch = async (criteria, pageNumber = 1, pageSize = 25, filterCatego
       },
     };
   }
+  
+  if (filterOutOrgNames && filterOutOrgNames.length > 0) {
```

Comment:
this can be sped up up just by doing the length test

Reply:
Done